pom.xml

```diff
@@ -288,7 +290,7 @@
 										<limit>
 											<counter>LINE</counter>
 											<value>MISSEDCOUNT</value>
-											<maximum>1400</maximum>
+											<maximum>1500</maximum>
```

Comment:
is it failing without this bump?

Reply:
Yes, the code formatting expanded all multi-line code into individual lines, but as you can see there were no real lines added.